### PR TITLE
Media Library: Display tooltips if transcoding is in progress

### LINKED ISSUE #6089: Video Optimization: Display tooltip for resource being transcoded
## Feature Description

In addition to the snackbar message, there should be a tooltip when hovering over a local media item in  the library that is currently being uploaded/processed.

Screenshot 2021-01-29 at 12 31 42

Same goes for the editor

**[Figma link](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=4846%3A254306)**

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by swissspidy**
## Context

We want to display a tooltip if video optimization is in progress for a given element/resource.

## Summary

This PR adds a tooltip for resources in the media library which are still being transcoded.

I was not able to do the same on the canvas, as neither the frame nor the display layer worked well with the tooltip's  absolute positioning.

## User-facing changes

New tooltip when hovering over media items in the library:

Screenshot 2021-03-01 at 18 01 40

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. Upload a video
2. Hover over video in media library
3. See a tooltip

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6089

### CODE REVIEW

**Comment by spacedmonkey** [comment]
Screenshot 2021-03-01 at 17 32 49
I uploaded a portrait video and had issue with menu overlaying the tooltip.

**Comment by swissspidy** [comment]
> Screenshot 2021-03-01 at 17 32 49
> I uploaded a portrait video and had issue with menu overlaying the tooltip.

That should be raised in a new ticket to update the tooltip component in the design system accordingly so it has higher z-index than the admin menu.

**Comment by spacedmonkey** [comment]
New issue created - https://github.com/google/web-stories-wp/issues/6581

File: assets/src/edit-story/components/library/panes/media/common/mediaElement.js [hunk 1]
```diff
@@ -92,28 +103,15 @@ const UploadingIndicator = styled.div`
...
- * @param {Object} param Parameters object
- * @param {number} param.index Index of the media element in the gallery.
- * @param {Object} param.resource Resource object
- * @param {number} param.width Width that element is inserted into editor.
- * @param {number} param.height Height that element is inserted into editor.
- * @param {string?} param.margin The margin in around the element
- * @param {Function} param.onInsert Insertion callback.
- * @param {string} param.providerType Which provider the element is from.
- * @return {null|*} Element or null if does not map to video/image.
- */
```
Comment (spacedmonkey) [inline assets/src/edit-story/components/library/panes/media/common/mediaElement.js:107]: Why remove these docs?

**Review by spacedmonkey — APPROVED** [review]
👍

**Pull request merged**